Zhongli.Bot/Bot.cs

```diff
@@ -55,9 +55,13 @@ private static void ContextOptions(DbContextOptionsBuilder optionsBuilder)
                 .AddUserSecrets<ZhongliContext>()
                 .Build();
 
-            optionsBuilder
-                .UseNpgsql(configuration.GetConnectionString(nameof(ZhongliContext)))
-                .UseLazyLoadingProxies();
+            var options = optionsBuilder.UseLazyLoadingProxies();
```

Comment:
You only need `optionsBuilder.UseLazyLoadingProxies();` here since it's a builder pattern, you don't need to reassign it.

Reply:
Done, I was just following your suggestion from discord. But you're right, we don't need to reassign.